drivers/espi/Kconfig

```diff
@@ -0,0 +1,98 @@
+# Kconfig - DMA configuration options
+#
+#
```

Comment:
remove

Reply:
It's does not need 2 #, but 1 is nice. comments don't have to be coalesced one after each other.